### ISSUE TITLE
Latest change to v1.3.2 breaks imports in ESM

### ISSUE THREAD

**Issue by ff6347** [marnusw/date-fns-tz#177]
I have been playing with pipedream for the last few hours and all over sudden my workflow fails. I get the following error:

```
ErrorDirectory import '/tmp/ee/node_modules/date-fns-tz/esm/_lib/newDateUTC' is not supported resolving ES modules imported from /tmp/ee/node_modules/date-fns-tz/esm/zonedTimeToUtc/index.js
```

I think it is this line that breaks the import.

https://github.com/marnusw/date-fns-tz/blob/4924dccc92cc421c955760acdd0e961bcc601c38/src/zonedTimeToUtc/index.js#L5

In all other imports you use the index.js at the end. Here not.

**Comment by marnusw** [comment]
Thank you, and sorry about that. I've released `1.3.3`.

**Comment by ff6347** [comment]
Thanks a lot for the quick action and don't apologize. It is me using your free software ;).
It is actually wired, that pipedream just takes the latests version. What could possibly go wrong?
